app/src/main/java/org/mozilla/fenix/library/recentlyclosed/RecentlyClosedAdapter.kt

```diff
@@ -9,17 +9,22 @@ import android.view.ViewGroup
 import androidx.recyclerview.widget.DiffUtil
 import androidx.recyclerview.widget.ListAdapter
 import mozilla.components.browser.state.state.recover.RecoverableTab
+import org.mozilla.fenix.library.SelectionHolder
```

Comment:
One thing you will realise, is that we've moved the SelectionHolder and SelectionInteractor to a higher package namespace in-order to re-use it in other components like tabs tray. Let's update these references.